src/gz.rs

```diff
@@ -862,6 +863,17 @@ impl Header {
     pub fn mtime(&self) -> u32 {
         self.mtime
     }
+
+    /// Returns the most recent modification time as a datetime if time stamp is available.
+    pub fn datetime(&self) -> Option<time::SystemTime> {
```

Comment:
Perhaps this could be called something like `mtime_as_datetime` to clarify that this is just reading the `mtime` field of the header?